### DISCUSSION TITLE
Perturbations of 64

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#463] [Discussion about coupled runs]
**Purpose: How robust is Lab Sea freeze in 64?**

64 froze around year 50. We have started a series of perturbed experiments to see how robust Lab Sea freeze in 64 is.

64e,f: start from 64 at year 43
64g,h: start from 64 at year 33
64intel: initial run with the new intel compiler

**Comment by adamrher** [comment]
![temp_tseries_SST](https://github.com/NCAR/amwg_dev/assets/29961476/1a0a9167-d850-46e7-9f26-97973e6d5e65)

**Reply by cecilehannay** [reply]
Thanks, @adamrher: I will continue: 64f, 64g, 64h and 64intel

**Comment by cecilehannay** [comment]
From Adam at [#455](https://github.com/NCAR/amwg_dev/discussions/455#discussioncomment-8151483 ): "Here's a current look. The only branch of 64 that froze over is 64e. Let's carry on with these experiments."

![temp_tseries_SST](https://github.com/NCAR/amwg_dev/assets/29961476/2a8a225f-ac7b-4e64-b0a6-49e36d179ada)

How long do you want to extent the perturbed 64 (64f, 64,g, 64h) ? Maybe to 100 years? So far, they are 80 years and I think we proved our point there is nothing intrinsic in 64.

**Reply by adamrher** [reply]
64 intel is a plugging along w/o any freeze. I don't really know what the point of that run is so I didn't include it.

**Reply by islasimpson** [reply]
This is really interesting.  Maybe rather than carry on with running these experiments our longer, it would be helpful to run a few more perturbations from the initial dates?  Then we could get a better picture of the probabilities of a freeze and maybe also start to understand what leads to the freeze by considering the members that freeze and those that don't.  The point has clearly been made that there's some randomness too it, but what the probabilities of a freeze versus no freeze are, is unclear.

**Reply by jedwards4b** [reply]
@adamrher that run is using the new ifx compiler instead of ifort.  ifx is scheduled to replace ifort later this year and we wanted to get a handle on any potential problems.

**Reply by adamrher** [reply]
plot updated with the intel run.
